infra/vf-mvp/index.ts

```diff
@@ -142,7 +142,7 @@ new aws.iam.RolePolicyAttachment(`${projectName}-fargate-task-role-sinuna-policy
 });
 
 // Fargate service
-new awsx.ecs.FargateService(
+var fargateService = new awsx.ecs.FargateService(
```

Comment:
This takes me back.. varmaan suoraan siirrytty tuoreena dotnet/C# projektista tähän riville? :D

Reply:
Niin ne syntaksit vähitellen sulautuu keskenään, tai vähintään jollain copilotin auto-syötöllä!